src/mock.ts

```diff
@@ -231,14 +252,12 @@ function addMockFunctionsToSchema({ schema, mocks = {}, preserveResolvers = fals
       mockResolver = mockType(field.type, typeName, fieldName);
     }
     if (!preserveResolvers || !field.resolve) {
-      // eslint-disable-next-line no-param-reassign
       field.resolve = mockResolver;
     } else {
       const oldResolver = field.resolve;
-      // eslint-disable-next-line no-param-reassign
-      field.resolve = (...args) => Promise.all([
-        mockResolver(...args),
-        oldResolver(...args),
+      field.resolve = (rootObject?: any, args?: { [key: string]: any }, context?: any, info?: GraphQLResolveInfo) => Promise.all([
```

Comment:
what do you mean by that? this is resolver signature..
am i missing something? or you mean the argument names?